### PR TITLE
Smart dashboard

### PR DESCRIPTION
**Pull request by Ariya3**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -41,7 +41,7 @@
     public static final int kDriveBaseCurrentLimit = 40;
 
     //Controller constants
-    public static final int kDBJoystickPort = 0;
+    //public static final int kDBJoystickPort = 0;
```
Comment (sarango2158) [inline src/main/java/frc/robot/Constants.java]: Can remove this line if commenting it out didn't break the build. Seems like a duplicate of kPortNumber

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -85,7 +86,9 @@ public void teleopInit() {
 
   // This function is called periodically during operator control. */
   @Override
-  public void teleopPeriodic() {}
+  public void teleopPeriodic() {
+    SmartDashboard.putNumber("Test", 100);
```
Comment (azvanderpas) [inline src/main/java/frc/robot/Robot.java]: Remove. this was test code.

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -5,9 +5,11 @@
 package frc.robot;
 
 import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
```
Comment (azvanderpas) [inline src/main/java/frc/robot/Robot.java]: Remove unused library

**Review by azvanderpas — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java [hunk 4]
```diff
@@ -5,9 +5,11 @@
...
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.IntakeSubsystem;
 import frc.robot.subsystems.HopperSubsystem;
```
Comment (azvanderpas) [inline src/main/java/frc/robot/Robot.java]: Remove unused

**Review by azvanderpas — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/Tabs/TabDriveBase.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        ShuffleboardTab dtTab = Shuffleboard.getTab("DriveBase");
+
+        sbLeftWheelSpeed = dtTab.add("Left Wheel Speed", 0).withSize(2, 2).withPosition(0, 0).getEntry();
+        sbRightWheelSpeed = dtTab.add("Right Wheel Speed", 0).withSize(2, 2).withPosition(3, 0).getEntry();
+
+        
+    }
+
+    public void periodic() {
+        // contantly update the board
```
Comment (tonyn10) [inline src/main/java/frc/robot/subsystems/Tabs/TabDriveBase.java:30]: spelled constantly wrong

**Review by sarango2158 — APPROVED** [review]

**Review by tonyn10 — APPROVED** [review]

File: src/main/java/frc/robot/Constants.java [hunk 6]
```diff
@@ -40,14 +40,6 @@
 
     public static final int kDriveBaseCurrentLimit = 40;
 
-    //Controller constants
-    public static final int kDBJoystickPort = 0;
-
-    public static final int kDBLeftJoystickAxisX = 0;
-    public static final int kDBLeftJoystickAxisY = 1;
```
Comment (Ariya3) [inline src/main/java/frc/robot/Constants.java:47]: kdb?

**Pull request merged**